--- osmo-0.4.4/data/sounds/Makefile.am~	2017-03-28 20:27:38.000000000 +0000
+++ osmo-0.4.4/data/sounds/Makefile.am	2022-03-15 11:58:00.466956111 +0000
@@ -1,5 +1,5 @@
 
-soundsdir = $(datadir)/sounds/osmo
+soundsdir = $(datadir)/osmo/sounds
 sounds_DATA = alarm.wav
 
 EXTRA_DIST = $(sounds_DATA) 
--- osmo-0.4.4/src/Makefile.am~	2020-07-12 08:13:46.000000000 +0000
+++ osmo-0.4.4/src/Makefile.am	2022-03-15 11:56:50.101668152 +0000
@@ -6,7 +6,7 @@
 VERSION_MICRO := $(shell echo $(VERSION) | awk -F "." '{print $$3}')
 AM_CPPFLAGS = -DREPO=$(ISREPO) -DREVISION=$(REVISION) -DLOCALEDIR=\"$(datadir)/locale\" -DDATADIR=\"$(datadir)\" \
 			  -DVERSION_MAJOR=\"$(VERSION_MAJOR)\" -DVERSION_MINOR=\"$(VERSION_MINOR)\" -DVERSION_MICRO=\"$(VERSION_MICRO)\" \
-			  -DSOUNDSDIR=\"$(datadir)/sounds\" @GTK_CFLAGS@ @XML_CPPFLAGS@ -Wall -DGDK_DISABLE_DEPRECATION_WARNINGS \
+			  -DSOUNDSDIR=\"$(datadir)/osmo/sounds\" @GTK_CFLAGS@ @XML_CPPFLAGS@ -Wall -DGDK_DISABLE_DEPRECATION_WARNINGS \
 			  -DICONSDIR=\"$(datadir)/icons\" -DPIXMAPSDIR=\"$(datadir)/pixmaps\" \
 			  -DG_DISABLE_CAST_CHECKS
 
